packages/venia-concept/client-schema.graphql

```diff
@@ -0,0 +1,19 @@
+extend type Query {
```

Comment:
Mind throwing a comment block above explaining who/what this file is for?

Reply:
For sure 👍  intend to collocate these closer to where they are defined, similar to what I did with the checkmo extension.